src/puppetlabs/puppetdb/config.clj

```diff
@@ -156,7 +156,7 @@
    :subname     (format "file:%s;hsqldb.tx=mvcc;sql.syntax_pgs=true" (io/file vardir "db"))})
 
 (defn validate-db-settings
-  "Throws a {:type ::configuration-error :message m} exception
+  "Throws a {:type ::cli-error :message m} exception
```

Comment:
Oh, and FWIW, I was imagining, for example, that a configuration-error (assuming that really meant config-file) might include :file and :line number members, and the catch might do something smart with them -- though perhaps you could also handle those as optional members of some shared map.